.eslintrc.cjs

```diff
@@ -21,7 +21,7 @@ module.exports = {
     'no-use-before-define': 'off',
     'import/no-named-as-default-member': 'off',
     'import/no-named-as-default': 'off',
-    'react/prop-types': 'warn',
+    'react/prop-types': 'off',
```

Comment:
But you should always validate your props to prevent inconsistencies while reusing your components. I would suggest that you revert this setting!